docs/TrafficSplitting.md

```diff
@@ -0,0 +1,163 @@
+# Traffic Splitting
+
+The purpose of this section of the kn documentation is to describe the traffic splitting operations with examples.
+
+The traffic splitting block for a Service may contain multiple targets. A target contains fields which can be operated by `kn` to define the
+behavior when a Service receives the traffic. One can define what percentage of the traffic should be routed to a particular Revision and also
+optionally create a custom URL for referenced Revision.
+
+`kn` client can work with following properties of a traffic target defined in traffic block of service.
+
+- **RevisionName**: Name of a specific the Revision to which to send a portion of traffic
+- **Tag**: Optionally used to expose a dedicated URL for referencing this target exclusively
+- **Percent**: Specifies percent of the traffic to this Revision
+- **LatestRevision**: Optionally used to indicate that the latest ready Revision of the Configuration should be used for this traffic target
+
+## Operations
+
+`kn` supports following operations on traffic block of a Service as part of `kn service update` command.
+
+### 1. Split Traffic:
+
+When you create a new Knative Service using `kn`, a new Revision is created with 100% traffic routing to it. With every update to the Configuration of the
```

Comment:
> With every update to the Configuration of the Service, a new Revision is created with Service route pointing all the traffic to the latest ready Revision by default.

Is this still correct? In case that users don't want to route traffic to the latest Revision (to be created by the update to the Configuration), should they run `kn service update` with `--traffic-latest 0` option?

I assume that a rule to the traffic routing never change unless traffic related options are set explicitly to `kn service update`, which will work as fail safe when they accidentally move to the invalid image or something like that.

Reply:
>  should they run `kn service update with --traffic-latest 0` option?

Yes, this should be a desired state `kn` should support.